tools/cldr-code/src/main/java/org/unicode/cldr/util/MatchValue.java

```diff
@@ -13,6 +13,9 @@
 import com.ibm.icu.text.UnicodeSet.SpanCondition;
 import com.ibm.icu.util.ULocale;
 import com.ibm.icu.util.VersionInfo;
+import com.vdurmont.semver4j.Semver;
```

Comment:
Hmm, any license issues for this code?

Reply:
no, it's MIT licensed, and pulled in as a dependency.